### ISSUE TITLE
Pre-work Checklist: Developer: Ronald

### ISSUE THREAD

**Issue by ronaldpaek** [hackforla/website#4074]
### Prerequisite
We are looking forward to having you on our team. Please make sure to attend the general Hack for LA onboarding to get the process started https://meetup.com/hackforla/events.

### Overview
As a new developer on the HfLA website team, fill in the following fields as you complete each onboarding item.

### Special Notes
1. It may take you a few weeks to finish this issue, because part of it is learning how to provide updates on issues that take more than a week to complete.    Please keep this issue open until you have been able to do all the steps.  
2. Usually we don't want to you have more than one issue assigned to you at a time, this issue is the exception, because it is instructions on how to work on other issues.
3. The action items listed below should mostly be worked on in a sequential order. However, you don't have to wait on one if you can proceed with the others, For instance, you don't have to wait for attending a weekly meeting before setting up your dev environment.  

### Action Items

- [x] Add yourself to the #hfla-site and #hfla-site-pr Slack channels
- [x] Share your GitHub handle and Gmail address in the hfla-site-onboarding slack channel so you can be added to the Google Drive (alternative to go to meeting and direct message a merge team member at the [meeting](https://github.com/hackforla/website/wiki/Meetings-and-Agendas
))
  - [x] To find contact information for the merge team members and technical leads, please take a look at our [Meet the Team wiki page](https://github.com/hackforla/website/wiki/Meet-the-Team)
- [x] Also, confirm with a merge team member or a technical lead that they have added you to the Google Calendar invites for our Zoom meetings
- [x] (Once added to the Drive) Add yourself to the [team roster](https://docs.google.com/spreadsheets/d/11u71eT-rZTKvVP8Yj_1rKxf2V45GCaFz4AXA7tS_asM/edit#gid=0)
- [x] After you have finished adding yourself to the roster, let a merge team member or a technical lead know you have added yourself to the roster and would like to be added to the `website-write` and `website` teams on GitHub
- [x] Once added to the website-write team:
  - [x] Self Assign this issue (gear in right side panel)
  - [x] Add the "role: front end" or "role: back end" or both label(s) to this issue and remove the "role missing" label (gear in right side panel)
  - [x] Add this issue to the Project Board under the Projects section (gear in right side panel)
- [x] Signup for a [Figma](https://Figma.com) account
- [x] Attend weekly team meetings:
  - [x] Developer (front-end/back-end) weekly team meeting, Tuesdays 7-8pm PST
  - [x] (Optional) Office Hours, Thursdays 7-8pm PST
  - [x] All team meeting (UX, Development, Product), Sunday 10am-12pm PST
- [x] Note: There are no meetings on the 1st-7th of every month.
- [x] Note regarding weekly team meeting requirements: All website team members are required to attend at least 1 team meeting in a week (held on Tuesdays, Thursdays and Sundays). In case, you are unable in any given week, you should reach out to the tech leadership team. Exceptions to this requirement may be provided on a case-by-case basis. Also, please let the tech leadership team know (through a slack message in the hfla-site channel as well as an @ mention in a comment of the issue that you would be working on) if you are planning to take a week off or a longer vacation.
- [x] Complete steps 1.1 - 1.6 in [Part 1: Setting up the development environment within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#part-1-setting-up-the-development-environment)
  - [x] OPTIONAL: If you run into any issues, use [4.1 How do I ask for help within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#41-what-do-i-do-if-i-need-help) as a visual example to get a hold of our [HfLA website leadership team](https://github.com/hackforla/website/projects/7#card-69730135). That way we can help you resolve any set up issues immediately.
- [x] Read section 2.1 - 2.4 in [Part 2: How the Website team works with GitHub issues within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#part-2-how-the-website-team-works-with-github-issues)
  - [x] Follow the steps in section [2.3 Where can I find GitHub issues to work on?](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#23-where-can-i-find-github-issues-to-work-on) to assign yourself your first issue under the Project Board's Prioritized Backlog column and use [2.7 Working on a Issue within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#27-working-on-an-issue) to start working on your issue
  - [x] Read [2.6 What to do when you need to stop mid issue](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#26-what-to-do-when-you-need-to-stop-mid-issue)
- [x] Once you take a good first issue, you will provide estimates and progress reports. Also, once you finish providing your "Estimate" action item below, please answer the question in the "Time spent so far" action item (also below).
   - [x] Estimate: Copy the below and put it in the "good first issue" that you picked.
 Check this box when you have completed this task so that we can identify if you understood the instructions and know what to do on all subsequent issues upon assignment. (Note: Please provide estimates on all the issues that you pick up, going forward.)
    ```
   Availability for this week:
   My estimated ETA for completing this issue:
    ```
   - [x] Time spent so far: Copy the question below into a new comment below (in this pre-work issue only) and answer it. This is just to get feedback on how long it took you to get to this point on this pre-work. There is no right or wrong answers. There is no judgement. It is ok if you take a long time or if you do it really fast or at any pace.
    ```
    How many hours did it take you to finish the pre-work up to and including adding your initial ETA and availability for your good first issue, including attending your first meetings?
    ```
   - [x] Progress Reports: Copy the below and put it in the issue once you have been assigned to the issue at least 5 days (we check weekly on Fridays), or sooner if you have something to report.  If you finish this issue before 5 days are reached, Yeah!!, do it on your next issue. **This update should be done every week for every issue that you are assigned to**. The checkbox here is meant for us to see if you understood the instructions when you end up doing your first weekly progress update.
    ```
    Provide Update
    1. Progress
    2. Blockers
    3. Availability
    4. ETA
    ```
- [x] Read sections 3.1.a - 3.1.c in [3.1 How to make a pull request](https://github.com/hackforla/website/blob/2e3c45fede99e61f479548ccca3f0dda83e351db/CONTRIBUTING.md#31-how-to-make-a-pull-request) to learn more about how to make a pull request (PR) for the issue that you are working on and how to make changes to your PR if changes are requested by the reviewer
- [x] Additional notes:
   - [x] Please don't forget to review pull requests whenever you are able to.  The [How to review Pull Requests](https://github.com/hackforla/website/wiki/How-to-review-pull-requests) guide will teach you how to review pull requests.
   - [x] Please work on only one issue at a time and wait until your pull request is merged before picking up another issue.
- [x] Read and understand how we progress through issues. Then, you can check this off.
Progress through issues with increasing complexity in the following order:
  - Good first issue (one per person)
  - Good second issue (one per person)
  - Small (one per person, with some exceptions, see below)
  - Medium (you can work on more than one medium issue, but only one at a time)
  - Large (you can work on more than one large issue, but only one at a time)
    - The reasons for this progression are:
      - The issues start out as being prescriptive and become less so as you gain more experience by working through increasingly complex issues.
      - We are trying to teach you the team methodology through the issues themselves.
      - It ensures you understand what we expect and the quality of contributions.
    - You can work on back-to-back small issues if it meets the following criteria:
      - You are learning something new and need to work on an issue of a lesser complexity
      - Special request by a lead or pm
- [x] Read the [Start Here - Developers](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=8583%3A0) in Figma
- [x] Go familiarize yourself with the [Hack for LA Design System page in Figma](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=3464%3A3)  (where you can see components and their SCSS classes)
- [x] Once all tasks are completed, close this issue.  This issue will remain open until you get through your first progress report.  Once you have done that, we are confident you know how to keep the momentum going on your issue and keep your team informed.

### What should I do if I have a question about an issue I'm working on, and I haven't gotten a response yet?
- First, you should post the question or blocker as a comment on your assigned issue, so it can be easily referred to in the next bullet points.
- Then, add the label "Status: Help Wanted" so other developers can see it and potentially help answer your question. In addition, you will still need to post a Slack message or bring it up in meeting so we know you need help; see below for how to do that.
- Also, you can post your question on the hfla-site slack channel and link the issue you're working on, so other developers can see and respond.
- Lastly, you can add the issue to the "Development team meeting discussion items" column of the Project Board so that it can be addressed in the next development meeting. Please bring it during the meeting that you need help.

### Resources/Instructions
- [Contributing.md - Hack for LA](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md)
- [GitHub Project Board - Hack for LA](https://github.com/hackforla/website/projects/7)
- [GitHub Project Board Prioritized Backlog highlighted and sorted by Front End](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+front+end%22)
- [GitHub Project Board Prioritized Backlog highlighted and sorted by Back End/DevOps](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+back+end%2Fdevops%22)
- [Figma - Hack for LA](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website)
- [Google Drive - Hack for LA website team](https://drive.google.com/drive/folders/1p76K0FgfiAWeIIEyoyJ_Iik8FVj8cBjT?usp=sharing)
- [Agenda / Notes - Dev Team Tuesday meeting](https://github.com/hackforla/website/issues/2010)
- [Agenda / Notes - All Team meeting](https://github.com/hackforla/website/issues/2027)
- [How to review Pull Requests](https://github.com/hackforla/website/wiki/How-to-review-pull-requests)

**Comment by ronaldpaek** [comment]
How many hours did it take you to finish the pre-work up to and including adding your initial ETA and availability for your good first issue, including attending your first meetings?

It took me about 3 hours for the onboarding meeting, and it took me about another 3 hours to finish reading over the checklist setting up my environment and finding a good first issue and making the changes, and submitting a PR. 

I have a question in terms of availability. Do we answer that in hours available per week or what days we are available to work on the issue? and as far as ETA that is a date estimate right not how many hours / days from now but rather a the date it should be completed as well?

Thank you,
Ronald

**Comment by ronaldpaek** [comment]
Provide Update
1. Made a pull request for first issue
2. No blockers
3. 6 hours
4. 03/02/23

**Comment by ExperimentsInHonesty** [comment]
Hi @ronaldpaek  if you have not given up on us yet and are still waiting on a [medium front end issue](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+front+end%22+label%3A%22complexity%3A+medium%22#column-7198257). There are 4 of them now available

**Comment by ronaldpaek** [comment]
@bonniewolfe, sorry about that. I'm going to need a week off. I will be back next week, I got into a car accident last Friday, and I need to get my car fixed and some other errands to take care of. Will I be able to resume then? 

And excellent Hoping to tackle some medium size issues when I come back. Thank you!

**Comment by ExperimentsInHonesty** [comment]
@ronaldpaek sorry to hear about your accident!  I'll stick this issue into the Questions / In Review column, so you don't get the 2-week inactive notices.  Move it back to in progress when you are ready.  We will check in April 9th, if we have not heard from you.

**Comment by ronaldpaek** [comment]
Thank you!

**Comment by ExperimentsInHonesty** [comment]
@ronaldpaek hope are feeling better and you car got all fixed up.  Are you back?  I am throwing this issue back into the in progress.  Let us know if you have left the team permanently.

**Comment by ronaldpaek** [comment]
@ExperimentsInHonesty thank you I am feeling better now and I ended up moving during that time, my apologies for not removing myself from the previous task I assigned myself. I can begin starting to work on tasks again start this week.

**Comment by ronaldpaek** [comment]
Progress: "I have identified which md files use the tools variable"
Blockers: "I am reading docs and videos to understand Jekyll liquid better."
Availability: "6/13/23"
ETA: "6/17/23"

**Comment by HackforLABot** [comment]
## Activity Log: ronaldpaek
### Repo: https://github.com/hackforla/website

#####  ⚠ Important note: The bot updates this comment automatically - do not edit

- ronaldpaek opened: https://github.com/hackforla/website/issues/4074 at 02/28/2023, 08:34 PM PST
- ronaldpaek assigned: https://github.com/hackforla/website/issues/4074 at 03/01/2023, 01:44 PM PST
- ronaldpaek assigned: https://github.com/hackforla/website/issues/4037 at 03/02/2023, 04:34 PM PST
- ronaldpaek opened: https://github.com/hackforla/website/pull/4093 at 03/02/2023, 06:31 PM PST
- ronaldpaek commented on: https://github.com/hackforla/website/issues/4074#issuecomment-1452873712 at 03/02/2023, 06:42 PM PST
- ronaldpaek commented on: https://github.com/hackforla/website/issues/4037#issuecomment-1452874654 at 03/02/2023, 06:42 PM PST
- ronaldpaek commented on: https://github.com/hackforla/website/issues/4037#issuecomment-1452884354 at 03/02/2023, 06:57 PM PST
- ronaldpaek commented on: https://github.com/hackforla/website/issues/4074#issuecomment-1453047813 at 03/02/2023, 10:33 PM PST
- ronaldpaek closed as completed: https://github.com/hackforla/website/issues/4074#event-8656637866 at 03/02/2023, 10:34 PM PST
- ronaldpaek commented on: https://github.com/hackforla/website/pull/4093#issuecomment-1454307201 at 03/03/2023, 04:51 PM PST
- ronaldpaek assigned: https://github.com/hackforla/website/issues/3973#issuecomment-1429299384 at 03/03/2023, 08:59 PM PST
- ronaldpaek commented on: https://github.com/hackforla/website/issues/3973#issuecomment-1454489257 at 03/03/2023, 09:43 PM PST
- ronaldpaek opened: https://github.com/hackforla/website/pull/4108 at 03/03/2023, 09:53 PM PST
- ronaldpaek PR was merged: https://github.com/hackforla/website/pull/4093#event-8666406999 at 03/04/2023, 08:32 AM PST
- ronaldpaek issue closed by PR 4093: https://github.com/hackforla/website/issues/4037#event-8666407035 at 03/04/2023, 08:32 AM PST
- ronaldpaek commented on: https://github.com/hackforla/website/issues/3973#issuecomment-1455181961 at 03/05/2023, 11:31 AM PST
- ronaldpaek commented on: https://github.com/hackforla/website/pull/4108#issuecomment-1456021488 at 03/06/2023, 04:11 AM PST
- ronaldpaek commented on: https://github.com/hackforla/website/pull/4108#issuecomment-1456034809 at 03/06/2023, 04:19 AM PST
- ronaldpaek commented on: https://github.com/hackforla/website/pull/4108#issuecomment-1464777957 at 03/10/2023, 05:55 PM PST
- ronaldpaek commented on: https://github.com/hackforla/website/pull/4108#issuecomment-1465153235 at 03/12/2023, 03:30 AM PDT
- ronaldpaek commented on: https://github.com/hackforla/website/pull/4108#issuecomment-1466980667 at 03/13/2023, 02:21 PM PDT
- ronaldpaek commented on: https://github.com/hackforla/website/pull/4108#issuecomment-1469183770 at 03/14/2023, 07:04 PM PDT
- ronaldpaek PR was merged: https://github.com/hackforla/website/pull/4108#event-8771055227 at 03/16/2023, 12:00 PM PDT
- ronaldpaek issue closed by PR 4108: https://github.com/hackforla/website/issues/3973#event-8771055561 at 03/16/2023, 12:00 PM PDT
- ronaldpaek commented on: https://github.com/hackforla/website/issues/4074#issuecomment-1483862269 at 03/25/2023, 09:22 AM PDT
- ronaldpaek commented on: https://github.com/hackforla/website/issues/4074#issuecomment-1484550015 at 03/26/2023, 11:08 PM PDT
- ronaldpaek assigned: https://github.com/hackforla/website/issues/4226 at 04/05/2023, 03:04 PM PDT
- ronaldpaek commented on: https://github.com/hackforla/website/issues/4226#issuecomment-1498217625 at 04/05/2023, 03:06 PM PDT
- ronaldpaek commented on: https://github.com/hackforla/website/issues/4226#issuecomment-1498220397 at 04/05/2023, 03:08 PM PDT
- ronaldpaek commented on: https://github.com/hackforla/website/issues/4226#issuecomment-1513755920 at 04/18/2023, 01:27 PM PDT
- ronaldpaek commented on: https://github.com/hackforla/website/issues/4226#issuecomment-1513758500 at 04/18/2023, 01:30 PM PDT
- ronaldpaek unassigned: https://github.com/hackforla/website/issues/4226 at 05/14/2023, 07:23 PM PDT
- ronaldpaek commented on: https://github.com/hackforla/website/issues/4074#issuecomment-1567714844 at 05/29/2023, 08:38 PM PDT
- ronaldpaek commented on: https://github.com/hackforla/website/issues/4525#issuecomment-1582004382 at 06/08/2023, 12:06 AM PDT
- ronaldpaek assigned: https://github.com/hackforla/website/issues/4525 at 06/08/2023, 12:07 AM PDT
- ronaldpaek commented on: https://github.com/hackforla/website/issues/4074#issuecomment-1589608689 at 06/13/2023, 09:04 AM PDT
- ronaldpaek commented on: https://github.com/hackforla/website/issues/4525#issuecomment-1590195813 at 06/13/2023, 04:36 PM PDT
- ronaldpaek commented on: https://github.com/hackforla/website/issues/4525#issuecomment-1590363829 at 06/13/2023, 07:49 PM PDT
- ronaldpaek assigned: https://github.com/hackforla/website/issues/4776 at 06/14/2023, 01:38 PM PDT
- ronaldpaek commented on: https://github.com/hackforla/website/issues/4776#issuecomment-1591950748 at 06/14/2023, 01:41 PM PDT
- ronaldpaek commented on: https://github.com/hackforla/website/pull/4842#issuecomment-1593819759 at 06/15/2023, 03:54 PM PDT
- ronaldpaek opened: https://github.com/hackforla/website/issues/4859 at 06/19/2023, 02:03 PM PDT
- ronaldpaek opened: https://github.com/hackforla/website/issues/4860 at 06/19/2023, 03:39 PM PDT
- ronaldpaek opened: https://github.com/hackforla/website/issues/4861 at 06/19/2023, 03:43 PM PDT
- ronaldpaek commented on: https://github.com/hackforla/website/issues/4776#issuecomment-1597848001 at 06/19/2023, 03:46 PM PDT
- ronaldpaek closed as completed: https://github.com/hackforla/website/issues/4074#event-9574384182 at 06/19/2023, 08:08 PM PDT
- ronaldpaek submitted review: https://github.com/hackforla/website/pull/4842#pullrequestreview-1489427291 at 06/20/2023, 06:01 PM PDT
- ronaldpaek commented on: https://github.com/hackforla/website/pull/4866#issuecomment-1599919946 at 06/20/2023, 06:11 PM PDT
- ronaldpaek submitted review: https://github.com/hackforla/website/pull/4866#pullrequestreview-1489473873 at 06/20/2023, 06:23 PM PDT
- ronaldpaek commented on: https://github.com/hackforla/website/pull/4701#issuecomment-1600363727 at 06/21/2023, 12:53 AM PDT
- ronaldpaek assigned: https://github.com/hackforla/website/issues/4768 at 06/21/2023, 04:43 AM PDT
- ronaldpaek commented on: https://github.com/hackforla/website/issues/4768#issuecomment-1600685798 at 06/21/2023, 04:44 AM PDT
- ronaldpaek commented on: https://github.com/hackforla/website/issues/4861#issuecomment-1600695743 at 06/21/2023, 04:52 AM PDT
- ronaldpaek commented on: https://github.com/hackforla/website/issues/4768#issuecomment-1610332304 at 06/27/2023, 04:03 PM PDT
- ronaldpaek submitted review: https://github.com/hackforla/website/pull/4832#pullrequestreview-1502200677 at 06/27/2023, 08:41 PM PDT
- ronaldpaek opened: https://github.com/hackforla/website/issues/4891 at 06/27/2023, 09:53 PM PDT
- ronaldpaek opened: https://github.com/hackforla/website/issues/4892 at 06/27/2023, 10:00 PM PDT
- ronaldpaek opened: https://github.com/hackforla/website/issues/4893 at 06/27/2023, 10:27 PM PDT
- ronaldpaek opened: https://github.com/hackforla/website/issues/4894 at 06/27/2023, 10:30 PM PDT
- ronaldpaek opened: https://github.com/hackforla/website/issues/4895 at 06/27/2023, 10:32 PM PDT
- ronaldpaek commented on: https://github.com/hackforla/website/issues/4768#issuecomment-1615271710 at 06/30/2023, 04:23 PM PDT
- ronaldpaek commented on: https://github.com/hackforla/website/issues/4776#issuecomment-1615272817 at 06/30/2023, 04:26 PM PDT
- ronaldpaek issue closed as completed: https://github.com/hackforla/website/issues/4776#event-9712430111 at 07/03/2023, 11:19 AM PDT
- ronaldpaek unassigned: https://github.com/hackforla/website/issues/4768 at 07/03/2023, 01:29 PM PDT
- ronaldpaek commented on: https://github.com/hackforla/website/issues/4768#issuecomment-1619121258 at 07/03/2023, 01:31 PM PDT
- ronaldpaek commented on: https://github.com/hackforla/website/issues/4861#issuecomment-1619634504 at 07/04/2023, 12:04 AM PDT
- ronaldpaek commented on: https://github.com/hackforla/website/issues/4861#issuecomment-1619635733 at 07/04/2023, 12:05 AM PDT
- ronaldpaek commented on: https://github.com/hackforla/website/issues/4860#issuecomment-1619637377 at 07/04/2023, 12:06 AM PDT
- ronaldpaek assigned: https://github.com/hackforla/website/issues/4887 at 07/07/2023, 04:23 AM PDT
- ronaldpaek commented on: https://github.com/hackforla/website/issues/4887#issuecomment-1625269223 at 07/07/2023, 04:25 AM PDT
- ronaldpaek commented on: https://github.com/hackforla/website/issues/4887#issuecomment-1628143807 at 07/09/2023, 09:33 PM PDT
- ronaldpaek opened: https://github.com/hackforla/website/pull/4927 at 07/09/2023, 11:12 PM PDT
- ronaldpaek commented on: https://github.com/hackforla/website/pull/4927#issuecomment-1629531137 at 07/10/2023, 11:55 AM PDT
- ronaldpaek submitted review: https://github.com/hackforla/website/pull/4926#pullrequestreview-1526869514 at 07/12/2023, 10:33 AM PDT
- ronaldpaek submitted review: https://github.com/hackforla/website/pull/4926#pullrequestreview-1526870688 at 07/12/2023, 10:34 AM PDT
- ronaldpaek commented on: https://github.com/hackforla/website/pull/4937#issuecomment-1633466380 at 07/12/2023, 07:46 PM PDT
- ronaldpaek commented on: https://github.com/hackforla/website/pull/4937#issuecomment-1633470305 at 07/12/2023, 07:51 PM PDT
- ronaldpaek commented on: https://github.com/hackforla/website/pull/4937#issuecomment-1633568597 at 07/12/2023, 10:26 PM PDT
- ronaldpaek commented on: https://github.com/hackforla/website/pull/4937#issuecomment-1633649218 at 07/12/2023, 11:41 PM PDT
- ronaldpaek submitted review: https://github.com/hackforla/website/pull/4937#pullrequestreview-1527710722 at 07/12/2023, 11:42 PM PDT
- ronaldpaek submitted review: https://github.com/hackforla/website/pull/4937#pullrequestreview-1529505668 at 07/13/2023, 07:40 PM PDT
- ronaldpaek PR was merged: https://github.com/hackforla/website/pull/4927#event-9818979423 at 07/14/2023, 12:35 AM PDT
- ronaldpaek issue closed by PR 4927: https://github.com/hackforla/website/issues/4887#event-9818979667 at 07/14/2023, 12:35 AM PDT
- ronaldpaek commented on: https://github.com/hackforla/website/issues/4059#issuecomment-1636924714 at 07/15/2023, 05:12 PM PDT
- ronaldpaek opened: https://github.com/hackforla/website/pull/5023 at 07/18/2023, 10:24 PM PDT
- ronaldpaek commented on: https://github.com/hackforla/website/pull/5023#issuecomment-1641435680 at 07/18/2023, 10:26 PM PDT
- ronaldpaek opened: https://github.com/hackforla/website/pull/5024 at 07/18/2023, 10:53 PM PDT
- ronaldpaek commented on: https://github.com/hackforla/website/pull/5024#issuecomment-1641468053 at 07/18/2023, 11:04 PM PDT
- ronaldpaek commented on: https://github.com/hackforla/website/pull/5024#issuecomment-1641469084 at 07/18/2023, 11:05 PM PDT
- ronaldpaek commented on: https://github.com/hackforla/website/issues/4861#issuecomment-1641475094 at 07/18/2023, 11:11 PM PDT
- ronaldpaek commented on: https://github.com/hackforla/website/pull/5024#issuecomment-1641479355 at 07/18/2023, 11:14 PM PDT
- ronaldpaek commented on: https://github.com/hackforla/website/pull/5025#issuecomment-1641500001 at 07/18/2023, 11:35 PM PDT
- ronaldpaek submitted review: https://github.com/hackforla/website/pull/5025#pullrequestreview-1536352491 at 07/18/2023, 11:37 PM PDT
- ronaldpaek submitted review: https://github.com/hackforla/website/pull/5006#pullrequestreview-1536369876 at 07/18/2023, 11:51 PM PDT
- ronaldpaek commented on: https://github.com/hackforla/website/pull/5023#issuecomment-1641528530 at 07/19/2023, 12:02 AM PDT
- ronaldpaek commented on: https://github.com/hackforla/website/issues/4860#issuecomment-1641740154 at 07/19/2023, 02:23 AM PDT
- ronaldpaek submitted review: https://github.com/hackforla/website/pull/4945#pullrequestreview-1537293770 at 07/19/2023, 07:52 AM PDT
- ronaldpaek commented on: https://github.com/hackforla/website/pull/4945#issuecomment-1642295715 at 07/19/2023, 08:22 AM PDT
- ronaldpaek commented on: https://github.com/hackforla/website/pull/4945#issuecomment-1642303851 at 07/19/2023, 08:27 AM PDT
- ronaldpaek commented on: https://github.com/hackforla/website/pull/4945#issuecomment-1642322603 at 07/19/2023, 08:38 AM PDT
- ronaldpaek commented on: https://github.com/hackforla/website/pull/4945#issuecomment-1642324980 at 07/19/2023, 08:39 AM PDT
- ronaldpaek commented on: https://github.com/hackforla/website/pull/4945#issuecomment-1642371972 at 07/19/2023, 09:09 AM PDT
- ronaldpaek opened: https://github.com/hackforla/website/pull/5035 at 07/19/2023, 11:29 AM PDT
- ronaldpaek closed: https://github.com/hackforla/website/pull/5035#event-9866009475 at 07/19/2023, 11:29 AM PDT
- ronaldpaek commented on: https://github.com/hackforla/website/pull/5024#issuecomment-1644018574 at 07/20/2023, 07:21 AM PDT
- ronaldpaek submitted review: https://github.com/hackforla/website/pull/5034#pullrequestreview-1539407604 at 07/20/2023, 07:29 AM PDT
- ronaldpaek commented on: https://github.com/hackforla/website/pull/5024#issuecomment-1644057299 at 07/20/2023, 07:44 AM PDT
- ronaldpaek commented on: https://github.com/hackforla/website/pull/5024#issuecomment-1644059158 at 07/20/2023, 07:45 AM PDT
- ronaldpaek submitted review: https://github.com/hackforla/website/pull/4945#pullrequestreview-1539900294 at 07/20/2023, 11:57 AM PDT
- ronaldpaek commented on: https://github.com/hackforla/website/pull/5024#issuecomment-1646905168 at 07/23/2023, 10:51 AM PDT
- ronaldpaek submitted review: https://github.com/hackforla/website/pull/5049#pullrequestreview-1544402721 at 07/24/2023, 04:03 PM PDT
- ronaldpaek commented on: https://github.com/hackforla/website/pull/5049#issuecomment-1648738323 at 07/24/2023, 04:04 PM PDT
- ronaldpaek submitted review: https://github.com/hackforla/website/pull/5052#pullrequestreview-1544411008 at 07/24/2023, 04:16 PM PDT
- ronaldpaek commented on: https://github.com/hackforla/website/issues/4768#issuecomment-1649341150 at 07/25/2023, 01:12 AM PDT
- ronaldpaek opened: https://github.com/hackforla/website/issues/5126 at 08/01/2023, 01:01 AM PDT
- ronaldpaek commented on: https://github.com/hackforla/website/issues/5126#issuecomment-1665896614 at 08/04/2023, 09:39 AM PDT
- ronaldpaek closed as completed: https://github.com/hackforla/website/issues/5126#event-10011329850 at 08/04/2023, 09:39 AM PDT
- ronaldpaek commented on: https://github.com/hackforla/website/pull/5124#issuecomment-1672356996 at 08/09/2023, 05:25 PM PDT
- ronaldpaek submitted review: https://github.com/hackforla/website/pull/5124#pullrequestreview-1574240003 at 08/11/2023, 12:05 PM PDT
- ronaldpaek commented on: https://github.com/hackforla/website/pull/5235#issuecomment-1681604512 at 08/16/2023, 09:43 PM PDT
- ronaldpaek submitted review: https://github.com/hackforla/website/pull/5235#pullrequestreview-1584394280 at 08/18/2023, 04:24 AM PDT
- ronaldpaek commented on: https://github.com/hackforla/website/issues/5210#issuecomment-1684804851 at 08/18/2023, 09:45 PM PDT
- ronaldpaek submitted review: https://github.com/hackforla/website/pull/5249#pullrequestreview-1585531524 at 08/18/2023, 09:47 PM PDT
- ronaldpaek opened: https://github.com/hackforla/website/pull/5258 at 08/19/2023, 02:34 AM PDT
- ronaldpaek commented on: https://github.com/hackforla/website/pull/5258#issuecomment-1684910689 at 08/19/2023, 03:08 AM PDT
- ronaldpaek submitted review: https://github.com/hackforla/website/pull/5263#pullrequestreview-1593327572 at 08/24/2023, 03:59 AM PDT
- ronaldpaek commented on: https://github.com/hackforla/website/pull/5258#issuecomment-1692792060 at 08/24/2023, 10:42 PM PDT
- ronaldpaek submitted review: https://github.com/hackforla/website/pull/5339#pullrequestreview-1604381237 at 08/31/2023, 02:52 AM PDT
- ronaldpaek commented on: https://github.com/hackforla/website/pull/5434#issuecomment-1705265530 at 09/04/2023, 06:20 AM PDT
- ronaldpaek commented on: https://github.com/hackforla/website/pull/5422#issuecomment-1705268041 at 09/04/2023, 06:22 AM PDT
- ronaldpaek commented on: https://github.com/hackforla/website/pull/5417#issuecomment-1705269236 at 09/04/2023, 06:23 AM PDT
- ronaldpaek submitted review: https://github.com/hackforla/website/pull/5434#pullrequestreview-1618569616 at 09/08/2023, 11:22 PM PDT
- ronaldpaek commented on: https://github.com/hackforla/website/pull/5024#issuecomment-1716758313 at 09/12/2023, 05:23 PM PDT
- ronaldpaek submitted review: https://github.com/hackforla/website/pull/5434#pullrequestreview-1623712985 at 09/12/2023, 10:25 PM PDT
- ronaldpaek submitted review: https://github.com/hackforla/website/pull/5512#pullrequestreview-1625486494 at 09/13/2023, 02:33 PM PDT
- ronaldpaek submitted review: https://github.com/hackforla/website/pull/5702#pullrequestreview-1679754125 at 10/16/2023, 04:36 AM PDT
- ronaldpaek commented on: https://github.com/hackforla/website/pull/5877#issuecomment-1800857938 at 11/07/2023, 05:29 PM PST
- ronaldpaek submitted review: https://github.com/hackforla/website/pull/5877#pullrequestreview-1719173090 at 11/07/2023, 05:50 PM PST
- ronaldpaek submitted review: https://github.com/hackforla/website/pull/5877#pullrequestreview-1721253234 at 11/08/2023, 12:51 PM PST
